### PR TITLE
Visual fixes (startpage)

### PR DESCRIPTION
**Pull request by srudersdorf**
- changed hero image to SVG with embedded JPG so the non-image parts are transparent
- optimized typography/legibility 
- updated button style
- visually fixed coloured ticket boxes
- fixed font margins

Especially the header could use some further browser tests. Checked for Safari, Chrome, Firefox (all latest version)

### CODE REVIEW

File: public/_general.scss [hunk 1]
```diff
@@ -90,36 +100,60 @@ a {
   height: 90px;
 }
 
+#main #hero {
+  color: white;
```
Comment (jschirrmacher) [inline public/_general.scss]: Somehow on small screens, the changes cause the background to be too small for the hero text
Bildschirmfoto 2019-08-12 um 18 00 17

File: templates/index.mustache [hunk 2]
```diff
@@ -19,25 +19,25 @@
             Start-ups, Design
             Thinker, Agile und Lean Startup-Experten der Metropolregion Frankfurt Rhein Main zusammen.</p>
 
-        <a class="button turquoise with-arrow" target="_self" href="tickets">Jetzt Ticket kaufen</a>
+        <a class="button yellow big" target="_self" href="tickets">Jetzt Ticket kaufen</a>
```
Comment (jschirrmacher) [inline templates/index.mustache]: Why did you remove the icons? Jens won't like that change... 😏

File: templates/index.mustache [hunk 3]
```diff
@@ -19,25 +19,25 @@
...
     </div>
 
     <div class="container no-background">
        <div id="what-is-it" class="three-boxes">
             <div class="box magenta">
-                <h3><span class="circle">✓</span>Zuhören und praktisch Eintauchen</h3>
+                <h3>Zuhören und praktisch Eintauchen</h3>
```
Comment (jschirrmacher) [inline templates/index.mustache]: Same here

File: public/_button.scss [hunk 4]
```diff
@@ -41,28 +42,68 @@ a, button {
...
+  font-weight: normal;
+  font-size: 1rem;
+  line-height: 1.25;
+
+  strong {
+    font-size: 1.2rem;
+  }
+}
+
+.yellow {
```
[the remark this answers is not in the document]
Comment (srudersdorf) [reply public/_button.scss]: from what I´ve seen so far, that´s fine. `yellow` only applies rules on top of `box`.

File: public/_general.scss [hunk 5]
```diff
@@ -90,36 +100,60 @@ a {
...
+    @media (min-width: 950px) {
+      padding: 3vw 3em 1em;
+    }
+
+    @media (min-width: 1100px) {
+      padding-top: 8vw;
+    }
+
+
+    .no-background {
```
[the remark this answers is not in the document]
Comment (srudersdorf) [reply public/_general.scss]: thx... removed by accident. Pushed a fix

File: .gitignore [hunk 6]
```diff
@@ -10,3 +10,5 @@
 /config/migrationData_*.json
 /cypress/screenshots
 /cypress/videos
+*.map
```
[the remark this answers is not in the document]
Comment (srudersdorf) [reply .gitignore]: hm... shouldn´t have happened. Can´t see it here. However, they get created during the SASS rendering. So should be all fine

File: templates/index.mustache (hunk 2 above)
Comment (srudersdorf) [reply templates/index.mustache]: It's not that I don´t like them :) I took them out because the headline was flowing around the icon in a way that made the text less readable. 

Happy to take them back in if Jens comlains :) but would then suggest to put them on top of the headline

File: public/_general.scss (hunk 1 above)
Comment (srudersdorf) [reply public/_general.scss]: tried to solve this pragmatically. Pushed a fix

Comment (jschirrmacher) [reply public/_general.scss]: Still a problem on other pages like /tickets

File: templates/index.mustache (hunk 2 above)
Comment (jschirrmacher) [reply templates/index.mustache]: Maybe a topic for today's call?

**Comment by jschirrmacher** [comment]
I made a temporary fix for all pages which doesn't yet have the new design to enable merge of this PR. Pls have a look at https://github.com/jschirrmacher/xcamp/pull/7/commits/4fc6e02d6993590c805ae16129043aa782bcefd7 to see the changes.
